api/handlers.py

```diff
@@ -0,0 +1,45 @@
+from flask import jsonify
+import MetaTrader5 as meta
+from datetime import datetime
+
+def get_open_orders():
+    try:
+        tuple = meta.orders_get()
+        orders_arr = []
+        for o in tuple:
+            data = {
+                "ticket": o.ticket,
+                "time_setup": o.time_setup_msc,
+                "order_type": o.type,
+                "volume_current": o.volume_current,
+                "stop_loss": o.sl,
+                "take_profit": o.tp,
+                "price_open": o.price_open,
+                "price_current": o.price_current,
+                "symbol": o.symbol,
+                "comment": o.comment
+            }
+            orders_arr.append(data)
+        return jsonify({ "orders": orders_arr }), 200
+    except BaseException:
+        return jsonify({ "error": meta.last_error() }), 500
+
+def get_closed_orders():
```

Comment:
same as above

Reply:
Once you implement flask calling "get_closed_orders()" & "get_account_info()" from IAccount then this 'hanlders.py' file will become unnecessary

Reply:
I'm not sure I follow. Are you saying that the flask api call should be handled outside of an API handlers class? That does not sound right to me. I believe we should have all of the APIs live together and they can reference classes that already exist, if needed, but the API layer should be distinct.

Reply:
Here's an outline of the pattern I think we should follow: https://anujbhor.medium.com/exploring-the-power-of-3-tier-architecture-in-modern-software-development-226c9431e525
It makes sure that each layer has a role and it's straightforward to know where to look when something needs to be changed.